lib/omnibus/packagers/ips.rb

```diff
@@ -214,17 +214,32 @@ def write_transform_file
     end
 
     #
+    # The name of the project specific template if it exists
+    # The resource exists locally. For example for project omnibus-toolchain
+    # resource_path("#{safe_base_package_name}-symlinks.erb") #=>
+    # {"/path/to/omnibus-toolchain/resources/omnibus-toolchain/ips/omnibus-toolchain-symlinks.erb"}
+    # OR {"/path/to/omnibus-toolchain/resources/omnibus-toolchain/ips/symlinks.erb"}
+    #
+    # @return [String]
+    #
+    def symlinks_file
```

Comment:
[nit] If this only called to be passed to `resource_path`, and it's doing `resource_path` itself, it might make sense to just return either of the two `resource_path`s. 🤔 

But what is there is fine...